Sim/SimDelphesInterface/src/DelphesSimulation.h

```diff
@@ -70,45 +77,66 @@ class DelphesSimulation: public GaudiAlgorithm {
   // Finalize.
   virtual StatusCode finalize();
 
-  // Convert internal Delphes objects to FCC EDM objects - UPDATE
-  void  ConvertParticle(TObjArray *  Input, fcc::ParticleCollection * coll );
-  void  ConvertJet(     TObjArray *  Input, fcc::GenJetCollection   * coll );
-  void  ConvertMET(     TObjArray *  Input, fcc::ParticleCollection * coll );
-  void  ConvertHT(      TObjArray *  Input, fcc::ParticleCollection * coll );
-
 private:
 
   // Delphes detector card to be read in
-  std::string              m_DelphesCard;
-
-  // Output collections to be write out
-  std::vector<std::string> m_outCollections;
+  std::string            m_DelphesCard;
```

Comment:
This could also be done in another iteration, but since I see that you did some doxygen documentation of the class :-). Could you add some comments for the data and function members? Some names are not completely clear (e.g. the *OutArrays at first glance might look like your output not the Delphes output).